src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -147,6 +147,10 @@ class: "Symfony\Component\VarDumper\Tests\Caster\ReflectionCasterTest"
      */
     public function testGenerator()
     {
+        if (ini_get('xdebug.overload_var_dump') == 2) {
```

Comment:
sorry if the previous comment wasn't clear enough :)
this specific test should use extension_loaded as before (and the next check is now fine)